admin/code/CMSProfileController.php

```diff
@@ -59,6 +57,7 @@ public function canView($member = null) {
 		if(
 			!Permission::checkMember($member, "CMS_ACCESS_LeftAndMain")
```

Comment:
Technically, if the user has ANY CMS_ACCESS_* permission, they should be able to use this feature. It's still good to have this explicit permission, but unfortunately the permission model doesn't support nested permissions yet (e.g. CMS_ACCESS_CMSMain is a child of CMS_ACCESS).